headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -1023,13 +1039,18 @@ struct dungeon_generation_info {
     // 0x0: Set if the floor layout is guaranteed to be a Monster House, or the dungeon generation
     // algorithm fails
     bool force_create_monster_house;
-    undefined field_0x1;
-    undefined field_0x2;
+    // 0x1: Set if the locked door on the dungeon floor has already been open.
+    bool locked_door_opened;
+    // 0x2: Set if a kecleon shop was properly spawned.
+    bool kecleon_shop_spawned;
+    // 0x3: When a non-zero value, the one-room orb will fail.
     undefined field_0x3;
```

Comment:
Can we name this one?

Reply:
I wasn't exactly sure what to name it because I couldn't find where the value gets set, but I know it gets checked in the function DoMoveOneRoom. Did you have any suggestions?

Reply:
`unk_one_room_flag` sounds good to me